cores/esp8266/core_esp8266_version.h

```diff
@@ -0,0 +1,30 @@
+
+#ifndef __CORE_ESP8266_VERSION_H
+#define __CORE_ESP8266_VERSION_H
+
+#define CORE_ESP8266_MAJOR    (2)
+#define CORE_ESP8266_MINOR    (4)
+#define CORE_ESP8266_REVISION (2)
```

Comment:
There's partial overlap between these and the ones in core_version.h.

Can defines in core_version.h be written using these new ones?